doc/data/messages/p/positional-only-arguments-expected/bad.py

```diff
@@ -0,0 +1,8 @@
+import random
```

Comment:
I think we should link to pep570 in related. There's also a bunch of usable examples in pep570. I think when args are called item1/2/3 it shows they should be a list called 'items'. Or we could use various thing like fruit, vegetable, animal

Reply:
updated for an easier example

Reply:
done